README.md

```diff
@@ -212,14 +259,24 @@ If we store this schema against a resource, it will be used to perform a more th
 | 5   | 'E'      | 2017-11-01 | missing     |                |
 | 'a' | 'B'      | 21/03/2017 |             |                |
 
-With the extension enabled and configured, schemas can be attached to the `schema` field on resources via the UI form or the API. If present in a resource, they will be used when performing validation on the resource file.
+With the extension enabled and configured, schemas can be attached to the
+`schema` field on resources via the UI form or the API. If present in a
+resource, they will be used when performing validation on the resource file.
 
 
 ### Validation Options
 
-As we saw before, the validation process involves many different checks and it's very likely that what "valid" data actually means will vary across CKAN instances or datasets. The validation process can be tweaked by passing any of the [supported options](https://github.com/frictionlessdata/goodtables-py#validatesource-options) to Goodtables. These can be used to add or remove specific checks, control limits, etc.
+As we saw before, the validation process involves many different checks and
+it's very likely that what "valid" data actually means will vary across CKAN
+instances or datasets. The validation process can be tweaked by passing any of
+the [supported
+options](https://github.com/frictionlessdata/goodtables-py#validatesource-options)
```

Comment:
Update link, or document what's possible

Reply:
For now I think the best page is https://framework.frictionlessdata.io/docs/guides/validating-data.html